client-js/README.md

```diff
@@ -0,0 +1,170 @@
+
+Further information on the CEP-78 Enhanced NFT Standard itself can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+## Requirements
+
+The JavaScript client for CEP-78 requires the use of the CasperLabs SDK for JavaScript. Information on installation and use of the Casper JS SDK is available [here](https://github.com/casper-ecosystem/casper-js-sdk).
+
+It also requires the use of `ts-results`, available [here](https://github.com/vultix/ts-results).
+
+These dependencies can be installed using the following command in the `client-js` directory:
+
+```js
+npm i
+```
+
+## Installing a CEP-78 Contract using the JavaScript Client
+
+The `install` method crafts a [Deploy](https://docs.casperlabs.io/design/casper-design/#execution-semantics-deploys) using `InstallArgs` and sends that Deploy to the specified Casper network.
+
+```js
+
+public install(
+    args: InstallArgs,
+    paymentAmount: string,
+    deploySender: CLPublicKey,
+    keys?: Keys.AsymmetricKey[],
+    wasm?: Uint8Array
+  ) {
+    const wasmToInstall =
+      wasm || getBinary(`${__dirname}/../wasm/contract.wasm`);
+      
+
+    const runtimeArgs = RuntimeArgs.fromMap({
+      collection_name: CLValueBuilder.string(args.collectionName),
+      collection_symbol: CLValueBuilder.string(args.collectionSymbol),
+      total_token_supply: CLValueBuilder.u64(args.totalTokenSupply),
+      ownership_mode: CLValueBuilder.u8(args.ownershipMode),
+      nft_kind: CLValueBuilder.u8(args.nftKind),
+      json_schema: CLValueBuilder.string(JSON.stringify(args.jsonSchema)),
+      nft_metadata_kind: CLValueBuilder.u8(args.nftMetadataKind),
+      // two below can conflict
+      identifier_mode: CLValueBuilder.u8(args.identifierMode),
+      metadata_mutability: CLValueBuilder.u8(args.metadataMutability),
+    });
+
+    if (args.mintingMode !== undefined) {
+      runtimeArgs.insert("minting_mode", CLValueBuilder.u8(args.mintingMode));
+    }
+
+    if (args.allowMinting !== undefined) {
+      runtimeArgs.insert(
+        "allow_minting",
+        CLValueBuilder.bool(args.allowMinting)
+      );
+    }
+
+    if (args.whitelistMode !== undefined) {
+      runtimeArgs.insert(
+        "whitelist_mode",
+        CLValueBuilder.u8(args.whitelistMode)
+      );
+    }
+
+    if (args.holderMode !== undefined) {
+      runtimeArgs.insert("holder_mode", CLValueBuilder.u8(args.holderMode));
+    }
+
+    if (args.contractWhitelist !== undefined) {
+      const list = buildKeyHashList(args.contractWhitelist);
+      runtimeArgs.insert("contract_whitelist", CLValueBuilder.list(list));
+    }
+
+    if (args.burnMode !== undefined) {
+      const value = CLValueBuilder.u8(args.burnMode);
+      runtimeArgs.insert("burn_mode", CLValueBuilder.option(Some(value)));
+    }
+
+    return this.contractClient.install(
+      wasmToInstall,
+      runtimeArgs,
+      paymentAmount,
+      deploySender,
+      this.networkName,
+      keys || []
+    );
+  }
+
+```
```

Comment:
I wouldn't add this code anywhere in the tutorial as this is internal implementation details. You should use it from this file:
https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/js-client/client-js/e2e/install.ts#L19-L62

but please modify schema and arguments :)